src/bytes.rs

```diff
@@ -17,6 +18,10 @@ use pyo3::types::PyBytes;
 pub struct Bytes(Vec<u8>);
 
 impl Streamable for Bytes {
+    fn get_hash(&self, digest: &mut Sha256) {
```

Comment:
Now that I see it... maybe `update_digest`?

Reply:
yeah, I'll change it to `update_digest()`